plugins/vbe/example/com.Vbe.Board.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Richard Hughes <richard@hughsie.com> -->
+<component type="firmware">
+  <id>com.Vbe.Laptop.firmware</id>
+  <name>VBE BoardFirmware</name>
+  <summary>System firmware for a board, for use with VBE</summary>
+  <description>
+    <p>
+      The board can be updated using Verified Boot for Embedded (VBE).
+    </p>
+  </description>
+  <provides>
+    <!-- this is a suitable HWID, found using `fwupdmgr hwids` -->
+    <firmware type="flashed">bb3b05a8-ebef-11ec-be98-d3a15278be95</firmware>
```

Comment:
I think you want to include the A/B into the instance ID, no?

Reply:
What is A/B info? For now I have only implemented a simple update, not an A/B update

Reply:
Also do you really want to be targeting an update to A or B by the HWID?  I feel like that detail is only relevant when flashing, not when targeting.

Reply:
I don't think I understand what is meant by A or B.

Reply:
Slot A vs Slot B.  If you end up eventually splitting this into a two stage process across multiple boots (which is mentioned in the TODO section in README.md), the question was if there needs to have this information in the GUIDs.  I would argue no - the payload is the same no matter which slot you flash in, but it will come down to your (later) implementation for flashing both slots if that is true.

Reply:
OK. That info is particular to the system itself. The update is just the update, but fwupd will need to decide which slot to use.